.pre-commit-config.yaml

```diff
@@ -12,31 +12,6 @@ repos:
       - id: detect-private-key
       - id: check-shebang-scripts-are-executable
 
-  - repo: https://github.com/adrienverge/yamllint.git
-    rev: v1.32.0
-    hooks:
-      - id: yamllint
-        entry: yamllint --strict -c .hooks/linters/yamllint.yaml
-
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.3
-    hooks:
-      - id: prettier
-        files: \.(json)$
-        exclude: docs/container.md
-
-  - repo: https://github.com/jumanjihouse/pre-commit-hooks
-    rev: 3.0.0
-    hooks:
-      - id: script-must-have-extension
-        name: Ensure shell scripts end with .sh
-        types: [shell]
-        exclude: .bats
-      - id: shellcheck
```

Comment:
will figure out an action for this one later - we don't have that many shell scripts to fine to remove for now, more valuable in armory where there will be many more shell scripts